src/components/EmailSuccessPageContainer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Section } from "./Layout/Section";
+import { vars, mixin } from "../styles";
+import { rgba } from "polished";
+import { SvgIcon, SvgIconChecked } from "./UI/SvgIcon";
+
+export const SectionEmailSuccess = styled(Section)`
+  color: ${vars.greyDark};
+  padding-top: 60px;
+  padding-bottom: 60px;
+
+  h1 {
+    ${mixin.fontMontserratSemibold}
+    ${mixin.fontSize(36)}
+    text-transform: uppercase;
+  }
+
+  h5 {
+    ${mixin.fontMontserratSemibold}
+  }
+
+  .bg-gradient {
+    background-image: linear-gradient(to right, ${vars.blueLight} 0%, ${vars.pink} 100%);
+    color: ${vars.white};
+  }
+
+  .success {
+    background-color: ${vars.white};
+    max-width: 420px;
+    margin-left: auto;
+    margin-right: auto;
+    border-radius: 8px;
+    box-shadow: 0 8px 16px ${rgba(vars.black, 0.1)};
+    text-align: center;
+    overflow: hidden;
+  }
+
+  .tick {
+    border: 4px solid ${vars.white};
+    width: 90px;
+    height: 90px;
+    border-radius: 50%;
+    padding: 8px;
+  }
+
+  svg {
+    width: 100%;
+    height: 100%;
+    color: ${vars.white};
+  }
+`;
+
+export const EmailSuccessPageContainer = () => {
+  return (
+    <SectionEmailSuccess>
+      <div className="success">
+        <div className="px-3 py-5 bg-gradient">
+          <div className="row no-gutters align-items-center">
+            <div className="col-auto mx-auto">
+              <div className="tick mb-3">
+                <SvgIcon>
+                  <SvgIconChecked />
+                </SvgIcon>
+              </div>
+            </div>
+            <div className="col-12">
+              <h1 className="mb-0">Success</h1>
+            </div>
+          </div>
+        </div>
+        <div className="px-3 py-5">
+          <div className="row">
```

Comment:
nitpicking but is this useful ? (sounds like the div doing nothing)

Reply:
oh, i generally try to avoid editing any padding or margin to divs using bootstrap's classes of  `.row` and `.col`, this is to not interfere with the framework's grid layout workings. (bootstrap's `.row` has this -15px margin left and right, `.col` has the 15px padding left and right)

so when there is a need to give spacing, i would put the spacing tweaks as the parent of `.row` or the children of `.col` as necessary.

for this case the layout is simple, so maybe can be an exception to place the helper classes directly on `.row`, therefore effectively dropping the div. but this will need `.no-gutters` to negate the negative margins by `.row` to be "clean". i've updated this.